pkg/kubernetes/mtls.go

```diff
@@ -107,6 +110,31 @@ func ExportTrustChain(outputDir string) error {
 	return nil
 }
 
+// Check and warn if cert expiry is less than `warningDaysForCertExpiry` days.
+func CheckForCertExpiry() {
+	expiry, err := Expiry()
+	// The intent is to warn for certificate expiry only when it can be fetched.
+	// Do not show any kind of errors with normal command flow.
+	if err == nil {
+		daysRemaining := int(expiry.Sub(time.Now().UTC()).Hours() / 24)
+		if daysRemaining < warningDaysForCertExpiry {
+			warningMessage := ""
+			if daysRemaining == 0 {
+				warningMessage = "Root certificate of your kubernetes cluster expires today"
+			} else if daysRemaining < 0 {
+				warningMessage = "Root certificate your kubernetes cluster already expired"
+			} else {
+				warningMessage = fmt.Sprintf("Root certificate your kubernetes cluster expires in %v days", daysRemaining)
+			}
+			color.Set(color.FgHiYellow)
+			helpMessage := "Kindly renew to avoid any service interuptions."
+			message := fmt.Sprintf("%s. Expiry date: %s. \n %s", warningMessage, expiry.Format(time.RFC1123), helpMessage)
```

Comment:
nit: `WarningStatusEvent` takes variadic params and uses `fmt.Sprintf` internally. We can also do something like
```go
print.WarningStatusEvent(os.Stdout, "%s. Expiry date: %s.\n Kindly renew to avoid any service interruptions.", warningMessage, expiry.Format(time.RFC1123))
```

Reply:
The only benefit IMO is that this looks more idiomatic. I can see longer lines in our code-base, so might just want to keep it that way.  Or, we can introduce line breaks in the same statement (and where to put a line break can be subjective, for example, https://github.com/mvdan/gofumpt/issues/2). I would suggest this (the two function params are `io.Writer` and `...args`):
```go
print.WarningStatusEvent(os.Stdout, 
		"%s. Expiry date: %s.\n Kindly renew to avoid any service interruptions.", warningMessage, expiry.Format(time.RFC1123))
```